docs/standard/io/index.md

```diff
@@ -145,7 +145,7 @@ File and stream I/O (input/output) refers to the transfer of data either to or f
   
 -   The path-based compression types <xref:System.IO.Compression.ZipFile> and <xref:System.IO.Compression.ZipFileExtensions> are not available. Instead, use the types in the [Windows.Storage.Compression](http://msdn.microsoft.com/library/windows/apps/windows.storage.compression.aspx) namespace.  
   
- You can convert between .NET Framework streams and Windows Runtime streams, if necessary. For more information, see [How to: Convert Between .NET Framework Streams and Windows Runtime Streams](../../../docs/standard/io/how-to-convert-between-dotnet-streams-and-winrt-streams.md) or <xref:System.IO.WindowsRuntimeStreamExtensions>.  
+ You can convert between .NET Framework streams and Windows Runtime streams, if necessary. For more information, see [How to: Convert Between .NET Framework Streams and Windows Runtime Streams](../../../docs/standard/io/how-to-convert-between-dotnet-streams-and-winrt-streams.md) or `System.IO.WindowsRuntimeStreamExtensions`. <!--zz <xref:System.IO.WindowsRuntimeStreamExtensions>--> 
```

Comment:
I think that here we need to link to the MSDN page, https://msdn.microsoft.com/library/system.io.windowsruntimestreamextensions.aspx. Leave the comment here so that we can find it easily if we need to (and I suspect we probably will).